docs/src/components/Example.tsx

```diff
@@ -1,6 +1,8 @@
-export function Example({ children }) {
+export function Example({ children, className = '' }) {
```

Comment:
@ericclemmons added this so we can scope CSS styles to an example to make the actual example code more closely reflect the actual code.

Reply:
Nice, this is great!